docs/how-to/best-practices-for-nft-data.md

```diff
@@ -118,7 +118,9 @@ When your data is stored on IPFS, users can fetch it from any IPFS node that has
 
 If you're building a platform using IPFS for storage, it's important to pin your data to IPFS nodes that are robust and highly available, meaning that they can operate without significant downtime and with good performance. See our [server infrastructure documentation][docs-server-infra] to learn how [IPFS Cluster][ipfs-cluster] can help you manage your own cloud of IPFS nodes that coordinate to pin your platform's data and provide it to your users.
 
-If running your own infrastructure doesn't make sense for your platform or stage of growth, you can delegate this responsibility to a remote pinning service. Remote pinning services provide redundant, highly-available storage for your IPFS data, without any kind of "vendor lock-in". Because IPFS content is addressed by CID instead of location, you can switch between pinning services at any time, or migrate to your own infrastructure seamlessly as your platform grows.
+If running your own infrastructure doesn't make sense for your platform or stage of growth, you can delegate this responsibility to a remote pinning service. Remote pinning services like [Pinata](https://pinata.cloud) and [Eternum](https://www.eternum.io/) provide redundant, highly-available storage for your IPFS data, without any kind of "vendor lock-in". Because IPFS content is addressed by CID instead of location, you can switch between pinning services at any time, or migrate to your own infrastructure seamlessly as your platform grows. 
+
+As of April 2021, you can also use a new service from [Protocol Labs](https://protocol.ai) called [nft.storage](https://nft.storage) to get your data into IPFS, with long-term persistence backed by the decentralized [Filecoin](https://filecoin.io) storage network. To help foster the growth of the NFT ecosystem and preserve the new "digital commons" of cultural artifacts that NFTs represent, [nft.storage](https://nft.storage) provides _free_ storage and bandwidth for data that's referenced in public NFTs. Sign up for a free account at [https://nft.storage](https://nft.storage) and try it out!
```

Comment:
> provides _free_ storage and bandwidth for data that's referenced in public NFTs

@mikeal is that a fair way of putting things? I'm just looking for a way to say "this is for NFTs, not your vacation photos"... but I can also just say nothing and let them click through to the site.

Reply:
we can drop “referenced in” and just say “public NFT.” people might be confused about how we determine “referenced in” and we will be iterating on that over time